docs/index.html

```diff
@@ -233,7 +236,8 @@ <h5>Food and Agriculture</h5>
           </dd>
           <dt><dfn>FSMA Inspection</dfn></dt>
           <dd>
-            <a href="https://www.fda.gov/food/guidance-regulation-food-and-dietary-supplements/food-safety-modernization-act-fsma">Food
+            <a
```

Comment:
any reason for the whitespace changes outside of the core e-commerce section?  looks like a formatter may have kicked in or something.  Mind backing out all changes except for the direct changes to your section on e-commerce?  at that point i will go ahead and merge